docs/rules/comma-style.md

```diff
@@ -37,6 +37,7 @@ This rule also accepts an additional `exceptions` object:
     * `"ObjectExpression": true` ignores comma style in object literals
     * `"ObjectPattern": true` ignores comma style in object patterns of destructuring
     * `"VariableDeclaration": true` ignores comma style in variable declarations
+    * `"NewExpression": true` ignores comma style in the parameters of constructor expressions
```

Comment:
Could you add an example into `exceptions` section with this option?

Reply:
I could, but the other exceptions don't have examples, should `NewExpression` be different?